storio-basic-sample-app/src/main/java/com/pushtorefresh/storio/basic_sample/MainActivity.java

```diff
@@ -0,0 +1,80 @@
+package com.pushtorefresh.storio.basic_sample;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.widget.Toast;
+
+import com.pushtorefresh.storio.StorIOException;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import com.pushtorefresh.storio.sqlite.impl.DefaultStorIOSQLite;
+import com.pushtorefresh.storio.sqlite.operations.put.PutResults;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+public class MainActivity extends AppCompatActivity {
+
+    private StorIOSQLite storIOSQLite;
+
+    private RecyclerView recyclerView;
+    private TweetsAdapter tweetsAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        storIOSQLite = DefaultStorIOSQLite.builder()
```

Comment:
Lol, already wrote a comment about DI here and only then understood that it's basic sample :)